Capri.Web/src/components/CardsView.vue

```diff
@@ -0,0 +1,422 @@
+﻿<template>
```

Comment:
You should stick to one convention about naming files. If you used camelCase at the bottom, then stick to it. Don't mix conventions.

Reply:
Name corrected, sorry, I've just notice that when I add it to git it's ignored.